client/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.controller.js

```diff
@@ -0,0 +1,40 @@
+angular
+  .module('managerApp')
+  .controller('TelecomTelephonyGuidesCtrl', class TelecomTelephonyGuidesCtrl {
+    constructor($translate, TucToastError, TELEPHONY_GUIDES) {
```

Comment:
Both `$translate` and `TucToastError` aren't used.

```suggestion
    constructor(TELEPHONY_GUIDES) {
```